tests/unittests/tests-checksum/tests-checksum-fletcher32.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016 Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdint.h>
+
+#include "embUnit/embUnit.h"
+
+#include "checksum/fletcher32.h"
+
+#include "tests-checksum.h"
+
+static int calc_and_compare_checksum(const unsigned char *buf, size_t len,
+        uint32_t expected)
```

Comment:
Is there a coding convention rule about these?
I fixed it based on an assumption now..